packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Gantt/BasicGanttCharts/BarTypes.8.doc.mdx

```diff
@@ -0,0 +1,36 @@
+import GanttChart from '../../../example/Graphs/Gantt/BasicGanttCharts/BarTypes?dev-site-example';
+import PropsTable from '@cerner/terra-graphs/lib/components/Gantt/GanttChart?dev-site-props-table';
+import GraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Gnatt/ganttDefault.js?dev-site-codeblock';
+import DataObject from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Gantt/barTypesData.js?dev-site-codeblock';
+require('details-polyfill')
+
+# Simple Gantt
+This is a Gantt chart with different bar types.
```

Comment:
nit: lowercase G

Reply:
Since it is a noun I made it as Capital G, corrected in all doc files.

Reply:
My bad, you're right. It is a proper noun. I take it back.